lua/astrocommunity/colorscheme/catppuccin/init.lua

```diff
@@ -1,20 +1,32 @@
 return {
-  "catppuccin/nvim",
-  name = "catppuccin",
-  opts = {
-    integrations = {
-      alpha = true,
-      aerial = true,
-      dap = { enabled = true, enable_ui = true },
-      mason = true,
-      neotree = true,
-      notify = true,
-      nvimtree = false,
-      semantic_tokens = true,
-      symbols_outline = true,
-      telescope = true,
-      ts_rainbow = false,
-      which_key = true,
+  {
+    "catppuccin/nvim",
+    name = "catppuccin",
+    opts = {
+      integrations = {
+        alpha = true,
+        aerial = true,
+        dap = { enabled = true, enable_ui = true },
+        mason = true,
+        neotree = true,
+        notify = true,
+        nvimtree = false,
+        semantic_tokens = true,
+        symbols_outline = true,
+        telescope = true,
+        ts_rainbow = false,
+        which_key = true,
+      },
+    },
+  },
+  {
+    "nvim-telescope/telescope.nvim",
+    optional = true,
+    opts = {
+      highlight = {
+        enable = true,
+        additional_vim_regex_highlighting = false,
+      },
```

Comment:
Shouldn't these opts me extended? In case of an already existing user-config

Reply:
This does do the extension of the `opts` table here.